### PR TITLE
HTML-CSS-Popup — #done

### PR DESCRIPTION
**Pull request by mustbefail**
### HTML-CSS-Popup

[Demo](https://mustbefail.github.io/frontend-2021-homeworks/submissions/mustbefail/html-css-popup/) |
[Code base](https://github.com/mustbefail/frontend-2021-homeworks/tree/html-css-popup/submissions/mustbefail/html-css-popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by AMashoshyna** [comment]
Hi! Great job!

But you accidentally removed example files under `/ submissions/amashoshyna/`. Please restore them. You can do it, for example, by checking out the files from master branch (`git checkout  submissions/amashoshyna`) and commiting them

File: submissions/mustbefail/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+                  </li>
+                  <li>
+                    <a href="#">
+                      <img src="./svg/027-calendar.svg" alt="">
+                      item3
+                    </a>
+                  </li>
+                </ul>
+        
+
```
Comment (miroslav-kolomiets) [inline submissions/mustbefail/html-css-popup/index.html:138]: Unnecessary new lines.

File: submissions/mustbefail/html-css-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+              
+            </div>
+          </div>
+          
+        </li>
+        <li><a href="#" class="notify btn"><i class="fas fa-bell"></i></a></li>
+        <li><a href="#" class="user btn"><i class="fas fa-user"></i></a>
+      </li>
+
+
```
Comment (miroslav-kolomiets) [inline submissions/mustbefail/html-css-popup/index.html:147]: Unnecessary new lines.

File: submissions/mustbefail/html-css-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+        <li><a href="#" class="notify btn"><i class="fas fa-bell"></i></a></li>
+        <li><a href="#" class="user btn"><i class="fas fa-user"></i></a>
+      </li>
+
+      
+    </ul>
+    </nav>
+  </header>
+</body>
+</html>
```
Comment (miroslav-kolomiets) [inline submissions/mustbefail/html-css-popup/index.html:152]: Add new line.

File: submissions/mustbefail/html-css-popup/style.css [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+  color: #aaa;
+}
+
+a:focus {
+
```
Comment (miroslav-kolomiets) [inline submissions/mustbefail/html-css-popup/style.css:11]: Unnecessary new line.

File: submissions/mustbefail/html-css-popup/style.css [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+    border: 2px solid rgb(33, 180, 248, 0.5);
+
```
Comment (miroslav-kolomiets) [inline submissions/mustbefail/html-css-popup/style.css:13]: Unnecessary new line.

File: submissions/mustbefail/html-css-popup/style.css [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 1rem;
+
```
Comment (miroslav-kolomiets) [inline submissions/mustbefail/html-css-popup/style.css:21]: Unnecessary new line.

File: submissions/mustbefail/html-css-popup/style.css [hunk 7]
```diff
@@ -0,0 +1,195 @@
...
+.popup li{
+  list-style: none;
+  width: 80px;
+  height: 80px;
+  margin-bottom: 20px;
+}
+
+.popup li a {
+
```
Comment (miroslav-kolomiets) [inline submissions/mustbefail/html-css-popup/style.css:110]: Unnecessary new line.

File: submissions/mustbefail/html-css-popup/style.css [hunk 8]
```diff
@@ -0,0 +1,195 @@
...
+  box-sizing: border-box;
+  display: block;
+  padding: 5px;
+  text-align: center;
+  text-decoration: none;
+  color: #333;
+
```
Comment (miroslav-kolomiets) [inline submissions/mustbefail/html-css-popup/style.css:117]: Unnecessary new line.

File: submissions/mustbefail/html-css-popup/style.css [hunk 9]
```diff
@@ -0,0 +1,195 @@
...
+/* Handle */
+::-webkit-scrollbar-thumb {
+  -webkit-border-radius: 10px;
+  border-radius: 10px;
+  background: rgba(204, 204, 204, 0.8); 
+  -webkit-box-shadow: inset 0 0 6px rgba(136, 136, 136, 0.5); 
+}
+::-webkit-scrollbar-thumb:window-inactive {
+background: rgba(199, 198, 198, 0.3); 
+}
```
Comment (miroslav-kolomiets) [inline submissions/mustbefail/html-css-popup/style.css:195]: Add new line.

File: submissions/mustbefail/html-css-popup/style.css [hunk 10]
```diff
@@ -0,0 +1,195 @@
...
+  box-shadow: 1px 1px 43px inset rgba(75, 75, 75, 0.2);
+}
+
+
+#popup2 {
+  border-top: 1px solid rgb(240, 240, 240);
+  display: none;
+}
+
+
```
Comment (miroslav-kolomiets) [inline submissions/mustbefail/html-css-popup/style.css:174]: Unnecessary new line.

File: submissions/mustbefail/html-css-popup/style.css [hunk 11]
```diff
@@ -0,0 +1,195 @@
...
+  line-height: 3;
+  display: inline-block;
+  color: rgb(105, 105, 105);
+}
+
+.add-more a:hover {
+  box-shadow: 1px 1px 43px inset rgba(75, 75, 75, 0.2);
+}
+
+
```
Comment (miroslav-kolomiets) [inline submissions/mustbefail/html-css-popup/style.css:168]: Unnecessary new line.

File: submissions/mustbefail/html-css-popup/style.css [hunk 12]
```diff
@@ -0,0 +1,195 @@
...
+#popup2 {
```
Comment (miroslav-kolomiets) [inline submissions/mustbefail/html-css-popup/style.css:169]: It's better to use some more consistent class names, not popup popup2 popup3....

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]
Good job! But some minor code style and naming issues detected.

**Review by MaxKalashnyk — COMMENTED** [review]
Good job! 👍🏻 
It's not a blocker, but according to the comments of @miroslav-kolomiets about redundant spacings, you can use Prettier for automation formatting of your code.
And one comment about the font-awesome icons. It's not a blocker too. You can add just simple SVG icons for these 3 icons in order not to use the whole library for icons.

**Review by MaxKalashnyk — APPROVED** [review]

**Review by miroslav-kolomiets — APPROVED** [review]

**Comment by miroslav-kolomiets** [comment]
Good job!

**Pull request merged**